DOCKER.md

```diff
@@ -45,27 +46,26 @@ Execute the run script from the root directory, specifying your configuration as
 bash docker/run.sh config/pg-search default
 ```
 
-The application executes inside of a [Docker container](https://www.docker.com/resources/what-container/), which ensures a clean build and avoids any issues with existing environments and installed packages.
+The application executes inside a [Docker container](https://www.docker.com/resources/what-container/), which ensures a clean build and avoids any issues with existing environments and installed packages.
 
 When the container is set up, the `run.sh` launch script will open up a browser at `localhost:80`.
 
 ### 5. Done!
 
 Your application is now up and running in Docker. Try a few queries to see how it works.
 
-You can view your application's logs by running `docker logs -f vanswer`. You can stop the application and shut down the container with `docker stop vanswer`.
```

Comment:
Should the instructions here be reverted back to use "docker logs" and "docker stop"?